tests/testfiles/multiline.ipynb

```diff
@@ -21,6 +21,7 @@
     "         'spam': ['ham', 'eggs']} # oda:StructuredParameter\n",
     "\n",
     "opt: float|None = None\n",
+    "otheropt = 5 # oda:optional\n",
```

Comment:
is it also possible to use `typing.Optional`?

Reply:
Yes, there are 3 ways to express it in type annotation (using `Optional`, `Union`, `|`) plus `# oda:optional`

Reply:
Ah ok, you tested it before, not in this PR.